doc/import_production_data.md

```diff
@@ -0,0 +1,23 @@
+# Import Production Data
+
+The content_performance_manager database is no longer imported by default.
+
+To import production data, first either download the data as documented in 
+[the GOV.UK developer docs](https://docs.publishing.service.gov.uk/manual/get-started.html#7-import-production-data)
+
+OR
+
+Get someone to give you a database dump if you don't have access.
+
+Then from inside the development VM run
+
+```
+mac$ cd ~/govuk/govuk-puppet/development-vm/replication
+mac$ ./replicate-data-local.sh -s -r -q -t -m -e -d backups/YYYY-MM-DD/ -i "service-manual-publisher publishing_api link_checker_api email-alert-monitor content_data_admin content_tagger content_publisher organisations-publisher email-alert-api content-register policy-publisher content_audit_tool ckan local-links-manager"
+```
```

Comment:
Need to state what this command is doing and how it replaces the instructions within the tech docs.